README.md

```diff
@@ -166,6 +166,8 @@ Optional settings:
     - `NONE`: No text added. (Default.)
     - `CONFIGURATION`: The name of the configuration which the dependency belongs to (e.g. "
       implementation", "compileOnly", "jsMain").
+- **excludeConfigurationNames**:
+  - List of configuration name which should be ignored. e.g. "implementation", "testImplementation". Default is emptyList().
```

Comment:
```suggestion
  - List of configurations to be ignored. e.g. "implementation", "testImplementation". Default is emptyList().
```

Reply:
As https://github.com/iurysza/module-graph/pull/25#discussion_r1540970073, https://github.com/iurysza/module-graph/pull/25#discussion_r1540969395 says 
Maybe it should be "List of configuration names to be ignored" ??

Reply:
fixed https://github.com/iurysza/module-graph/commit/73b5e936e23403336309bc043e887c3c0bd3aa7d